packages/agent/src/certificate.test.ts

```diff
@@ -20,6 +19,24 @@ function pruned(str: string): ArrayBuffer {
   return fromHex(str);
 }
 
+function bufferEqualityTester(a: unknown, b: unknown): boolean | undefined {
```

Comment:
We already have `compare` and `bufEquals` utils in `packages/agent/src/utils/buffer.ts`